### ISSUE TITLE
chatoverlay throws npe

### ISSUE THREAD

**Issue by farukcankaya** [intercom/intercom-android#204]
## Version info
  - Intercom for Android version: 3.0.0-beta3
  - Android OS version: 4.4.2
  - Android Device: GT i9301q

## Logs
``````
Fatal Exception: java.lang.NullPointerException
       at io.intercom.android.sdk.preview.ChatOverlay.expandChat(ChatOverlay.java:171)
       at io.intercom.android.sdk.preview.ChatOverlay.moveBackward(ChatOverlay.java:91)
       at io.intercom.android.sdk.preview.NotificationManager.updateNotifications(NotificationManager.java:221)
       at io.intercom.android.sdk.preview.NotificationManager.displayPreviews(NotificationManager.java:162)
       at io.intercom.android.sdk.preview.NotificationManager.access$000(NotificationManager.java:35)
       at io.intercom.android.sdk.preview.NotificationManager$1.run(NotificationManager.java:67)
       at io.intercom.android.sdk.utilities.ViewUtils$1.onGlobalLayout(ViewUtils.java:25)
       at android.view.ViewTreeObserver.dispatchOnGlobalLayout(ViewTreeObserver.java:856)
       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2088)
       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1253)
       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6596)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:803)
       at android.view.Choreographer.doCallbacks(Choreographer.java:603)
       at android.view.Choreographer.doFrame(Choreographer.java:573)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:789)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5476)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
       at dalvik.system.NativeStart.main(NativeStart.java)
``````

**Comment by ronocod** [comment]
Hi @farukcankaya,

Thanks for reporting this, we're looking into fixing this now. It's quite hard to reproduce but we'll let you know here when we release a fix.

**Comment by ronocod** [comment]
Fixed in `3.0.0-beta4`.
